prisma/migrations/20240513040814_add_settings_table/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateEnum
+CREATE TYPE "ChannelSyncOptions" AS ENUM ('clientAndCompany');
+
+-- CreateTable
+CREATE TABLE "Settings" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "workspaceId" VARCHAR(32) NOT NULL,
+    "internalUserId" UUID NOT NULL,
+    "bidirectionalSlackSync" BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Can we add comments on columns? https://stackoverflow.com/a/32071293

Dont know if Prisma supports it in schema level. It will be helpful to know what the columns exactly are for.

can we rename the column to `syncStatus`? I find it more meaningful